src/sql/helpers.js

```diff
@@ -64,26 +64,44 @@ const jsonbObjectAgg = (aggregationColumnKey, nestedFields, aggregationJsonKey,
  * For this example:
  * - rootFields: ['id', 'name'],
  * - nestedFields: ['pipeline_arn', 'hash]
- * - aggregationColumnKey: 'pipeline_type'
- * - aggregationJsonKey: 'pipelines'
+ * - collapseKey: 'pipeline_type'
+ * - collapseKeyNewName: 'pipelines'
  *
  * Note it also camelcases all the keys returned
  */
-const aggregateIntoJson = (
+const collapseKeysIntoObject = (
   originalQuery,
   rootFields,
-  nestedFields,
-  aggregationColumnKey,
-  aggregationJsonKey,
+  fieldsToCollapse,
+  collapseKey,
+  collapseKeyNewName,
   sql,
 ) => (
   sql
     .select([
       ...rootFields,
-      jsonbObjectAgg(aggregationColumnKey, nestedFields, aggregationJsonKey, sql),
+      createCollapsedObject(collapseKey, fieldsToCollapse, collapseKeyNewName, sql),
     ])
     .from(originalQuery)
     .groupBy(rootFields)
 );
 
-module.exports = { aggregateIntoJson };
+const collapseKeyIntoArray = (
```

Comment:
rename `collapseKeyIntoArray` to `collapseKeysIntoArray` to match the naming of `collapseKeysIntoObject`

Reply:
In this case no, because we are collapsing only one key, while in the object we are collapsing many different keys